integrations/gcp/gcp_core/utils.py

```diff
@@ -119,7 +125,6 @@ def get_service_account_project_id() -> str:
         raise ValueError(
             f"Couldn't figure out the service account's project id. Error: {str(e)}"
         )
-    raise ValueError("Couldn't figure out the service account's project id.")
```

Comment:
Don't delete this

Reply:
It can be reached ?

Reply:
Just in case, so that we're not running into some None issues

Reply:
we have except Exception 😂